### ISSUE TITLE
legacy wrapper does not expose in-orbit coordinates pb meshes

### ISSUE THREAD

**Issue by bpablo** [phoebe-project/phoebe2#58]
phoebe1 only calculates a quarter of the star and therefore the values must be reflected over the appropriate axes. This is implemented incorrectly at present.

**Comment by aprsa** [comment]
Incorrectly how? discretize_using_wd() does reflect over the appropriate axes, and the wdeclipse branch has another version that sorts the surface elements by latitudes round and around.

**Comment by bpablo** [comment]
I didn't know about those. I meant purely in porting from phoebe1 to
phoebe2 and loading in a phoebe 1 mesh this isn't done (because I don't do
it). I suppose I could feed the phoebe1 values to those functions and it
will do all the work for me?

**Comment by kecnry** [comment]
No, its not that modular.  But if you look at phoebe.backend.potentials.discretize_wd_style in the master branch, it should be quite straightforward to replicate the logic in the legacy wrapper.

**Comment by aprsa** [comment]
Is this for the purpose of plotting the meshes? If so, I'd say this is the job for the plotting front-end, not the wrapper.

**Comment by kecnry** [comment]
Not just plotting, but also to inspect the mesh and compare with our WD-mesh implementation directly.

I think it's the wrapper's job to translate everything to match the format provided by phoebe2, at least as closely as possible.  So I'd be ok with only storing the quarter-hemisphere if we were going to do the same thing for our WD-mesh implementation, but that doesn't make sense anymore now that we care about the order of indices.

**Comment by bpablo** [comment]
This has been fixed, but still needs to be thoroughly tested.

**Comment by kecnry** [comment]
For 2.0 - let's just make sure that no incorrect meshes are exposed to the user.  We can deal with empty meshes for now.

Once that is confirmed, go ahead and change the milestone from 2.0 to future.

**Comment by bpablo** [comment]
protomeshes pulled from phoebe1 display correctly. Regular meshes lack coordinate information and a warning is given. This can be deferred.

**Comment by aprsa** [comment]
After having looked into this in detail, I don't think it's feasible at all. WD optimizes invisible elements away so I would need to rewrite a good chunk of `light.f` in order to accomplish this. So either we abandon it (my vote) or we implement xyz->uvw outside of the wrapper (could be convinced).
